<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSettingsWidget</class>
 <widget class="QWidget" name="DatabaseSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,5,1">
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="2" colspan="2">
        <widget class="QComboBox" name="kdfComboBox"/>
       </item>
       <item row="0" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dbNameLabel">
         <property name="text">
          <string>Database name:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="historyMaxSizeCheckBox">
         <property name="text">
          <string>Max. history size:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="kdfLabel">
         <property name="text">
          <string>Key derivation function:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="historyMaxItemsCheckBox">
         <property name="text">
          <string>Max. history items:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="dbNameEdit"/>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QSpinBox" name="historyMaxItemsSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>2000000000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="defaultUsernameLabel">
         <property name="text">
          <string>Default username:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="dbDescriptionEdit"/>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QSpinBox" name="historyMaxSizeSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> MiB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2000000000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QCheckBox" name="recycleBinEnabledCheckBox">
         <property name="text">
          <string>Use recycle bin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="defaultUsernameEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dbDescriptionLabel">
         <property name="text">
          <string>Database description:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QComboBox" name="algorithmComboBox"/>
       </item>
       <item row="6" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="algorithmLabel">
         <property name="text">
          <string>Algorithm:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbNameEdit</tabstop>
  <tabstop>dbDescriptionEdit</tabstop>
  <tabstop>defaultUsernameEdit</tabstop>
  <tabstop>recycleBinEnabledCheckBox</tabstop>
  <tabstop>historyMaxItemsCheckBox</tabstop>
  <tabstop>historyMaxItemsSpinBox</tabstop>
  <tabstop>historyMaxSizeCheckBox</tabstop>
  <tabstop>historyMaxSizeSpinBox</tabstop>
  <tabstop>algorithmComboBox</tabstop>
  <tabstop>kdfComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
